genisys/main.py

```diff
@@ -15,10 +17,21 @@ def validate(modules):
 
 def install_config(file, root="/"):
     print(f"Installing config file: {file} with root at {root}")
-    ps.install(root)
-    nt.install(root)
-    net.install(root)
-    kp.install(root)
+    # netplan
+    netplan = net.Netplan(file)
```

Comment:
We've got this being defined in both `run` and this function. Remember DRY, can we either pass the modules instantiated in `run` into this function as arguments or otherwise define them as globals?